tests/integration/single-trust-zone/test.sh

```diff
@@ -80,6 +80,39 @@ function wait_for_pong() {
   return 1
 }
 
+function show_workload_status() {
+  POD_NAME=$(kubectl get pods -l app=ping-pong-client \
+    -n $NAMESPACE_POLICY_NAMESPACE \
+    -o jsonpath='{.items[0].metadata.name}' \
+    --context $K8S_CLUSTER_CONTEXT)
+  WORKLOAD_STATUS_RESPONSE=$(./cofidectl workload status --namespace $NAMESPACE_POLICY_NAMESPACE \
```

Comment:
Is there anything we can use for a positive match? Otherwise we need to be aware of all possible errors here.

Reply:
We could always just check for the presence of this [string](https://github.com/cofide/cofidectl-debug-container/blob/55413bd8fac91fe521fdd796cb710cacbf10b8b8/cmd/main.go#L84), if we are happy for that to be enough? I opted to check for the errors as that felt a bit more comprehensive.

Reply:
I think that's a reasonable check.

Reply:
Okay I'll check for that and remove the checks for each of the error cases.